external/llvm-project/mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -1653,7 +1654,7 @@ def Tosa_ReverseOp: Tosa_Op<"reverse", [
 
   let hasFolder = 1;
   let hasVerifier = 1;
-  
+
```

Comment:
please remove spurious unncessary changes to external/ which will affect upstream merges.

Reply:
Trailing whitespace, the bane of my existence.